### PR TITLE
Migration from React Native repo

### PR DESCRIPTION
**Pull request by michalchudziak**
This PR introduces `@react-native-community/slider` as a separate component with a preserved history from `react-native` repo, it also includes the following changes:

* Renamed the native component to `RNCSlider` which follows the pattern from other components/packages (`WebView`, `NetInfo`)
* Placed the Android code into the `com.reactnativecommunity.slider` package which follows the pattern from other components/packages (`WebView`, `NetInfo`)
* Added an `example` project with the structure, that allows developing the module within the `example` file, without the need of reinstalling the app
* Contents of an example app are based on `RNTester` files

Screenshots:
Android:
image
image
iOS:
image
image

@cpojer Let me know if I should change something 😁

### CODE REVIEW

File: example/App.js [hunk 1]
```diff
@@ -10,6 +10,8 @@
 import React, {Component} from 'react';
 import {Platform, StyleSheet, Text, View} from 'react-native';
 
+const { examples } = require('react-native/RNTester/js/SliderExample');
```
Comment (cpojer) [inline example/App.js]: Can you get rid of the spaces here?

File: example/App.js [hunk 2]
```diff
@@ -8,9 +8,8 @@
  */
 
 import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
-
-const { examples } = require('react-native/RNTester/js/SliderExample');
+import {Platform, StyleSheet, Text, ScrollView, View} from 'react-native';
+import { examples } from './SliderExample';
```
Comment (cpojer) [inline example/App.js]: or here, actually!

**Comment by cpojer** [comment]
(Just added you as an owner to this repo so you can take merge PRs :) )

**Comment by michalchudziak** [comment]
Thanks!! For some reason I don't see an option to merge this PR :(

**Pull request merged**